common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -30,6 +32,28 @@ innovation_elephantry = {
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+innovation_kodo = {
```

Comment:
Mushans are worth adding. They're like kodos, but live in Pandaria.
https://wowpedia.fandom.com/wiki/Mushan

Reply:
👌 can I add them to ideas sheet and add them on another cultural maas addition?

Reply:
Mmmkay.